.yeoman/templates/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@bbc/psammead-<%= name %>",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "description": "<%= description %>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-<%= name %>"
+  },
+  "author": {
+    "name": "<%= authorName %>",
+    "email": "<%= authorEmail %>"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-<%= name %>/README.md",
+  "dependencies": {
+    "react": "<%= dependencies.react %>"
```

Comment:
Don't we have `react` as a `peerDependency`?

Reply:
![](https://media.giphy.com/media/xT5LMwv2kxT6RjODi8/giphy.gif)